prefs.js

```diff
@@ -0,0 +1,76 @@
+const { Adw, Gio, GLib, Gtk } = imports.gi;
+
+const ExtensionUtils = imports.misc.extensionUtils;
+const Me = ExtensionUtils.getCurrentExtension();
+
+/**
+ * Like `extension.js` this is used for any one-time setup like translations.
+ *
+ * @param {ExtensionMeta} meta - An extension meta object, described below.
+ */
+function init(meta) {
+    console.debug(`initializing ${Me.metadata.name} Preferences`);
+}
+
+/**
+ * This function is called when the preferences window is first created to build
+ * and return a GTK widget.
+ *
+ * As of GNOME 42, the preferences window will be a `Adw.PreferencesWindow`. The
+ * widget returned by this function will be added to an `Adw.PreferencesPage` or
+ * `Adw.PreferencesGroup` if necessary.
+ *
+ * @returns {Gtk.Widget} the preferences widget
+ */
+function buildPrefsWidget() {
```

Comment:
Looks like this is for GNOME < 42 and it'll show just a label with the extension name, but it doesn't matter, I think we can remove this function completely.